ocs_ci/deployment/deployment.py

```diff
@@ -336,6 +345,42 @@ def deploy_ocs(self):
         logging.info(f"Out of the WA cmd {wa_cmd} is: {out}")
         # end of WA
 
+        if config.ENV_DATA.get('monitoring_enabled') and config.ENV_DATA.get('persistent-monitoring'):
```

Comment:
Do you know if this is going to be handled by olm ocs operator? If so, this should be skipped in case of deploy via operator.  For example add also this to the condition:  https://github.com/red-hat-storage/ocs-ci/pull/479/files#diff-153ccf5916ca230718b5455c349afbcdR286

Reply:
AFAIK, only monitoring setup will be enabled by default not it's persistent storage